dali/python/nvidia/dali/triton.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Yeah, this could be also. @banasraf , what's your opinion?

Maybe we should think about the reversed situation? It seems to me that it'd be better to import frameworks like
```python
import nvidia.dali.mxnet
import nvidia.dali.tf
```

Reply:
For now, for the sake of consistency I would make it `nvidia.dali.plugin.triton`. If we decide to move framework integrations to the `nvidia.dali` level, we can do it later for all of them

Reply:
Done